api/api.go

```diff
@@ -17,6 +17,7 @@ type ClientService interface {
 	SyncNotify(flux.InstanceID) error
 	SyncStatus(flux.InstanceID, string) ([]string, error)
 	UpdatePolicies(flux.InstanceID, flux.PolicyUpdates) (job.ID, error)
+	LogEvent(inst flux.InstanceID, e flux.Event) error
```

Comment:
I agree, and had put it there, but the method is implemented as part of the api client. Will have a think about it. We could add a `fluxctl add-event -m "incident with the frobulator"` for user-custom events. :sparkles: